components/infobox/wikis/tft/infobox_person_player_custom.lua

```diff
@@ -29,6 +29,8 @@ local ROLES = {
 	talent = {category = 'Talents', value = 'Talent', type = 'talent'},
 	manager = {category = 'Managers', value = 'Manager', type = 'staff'},
 	producer = {category = 'Producers', value = 'Producer', type = 'talent'},
+	organizer = {category = 'Tournament Organizers', value = 'Organizer', type = 'staff'},

```

Comment:
```suggestion
	organizer = {category = 'Tournament Organizer', value = 'Organizer', type = 'staff'},
```
make the categories singular to fix https://discord.com/channels/93055209017729024/268719633366777856/1084647601527345192